intro-Linux/ch_uploading_files.tex

```diff
@@ -59,3 +59,41 @@ \section{Editing with \texttt{nano}}
 
 More advanced editors (beyond the scope of this page) are \verb|vim| and \verb|emacs|.
 A simple tutorial on how to get started with \verb|vim| can be found at \url{https://www.openvim.com/}.
+
+\section{Copying faster with \texttt{rsync}}
+\hypertarget{sec:rsync}{}
+
+\verb|rsync| is a fast and versatile copying tool. It can be much faster than \verb|scp|
+when copying large datasets. It's famous for its ``delta-transfer algorithm'', which
+reduces the amount of data sent over the network by only sending the differences between
+files.
+
+You will need to run \verb|rsync| from a computer where it is installed. Installing
+\verb|rsync| is the easiest on Linux: it comes pre-installed with a lot of distributions.
+
+For example, to copy a folder with lots of CSV files:
+
+\begin{prompt}
+%\shellcmd{rsync -rzv testfolder \userid{}@\loginnode{}:data/}
+\end{prompt}
+
+will copy the folder \verb|testfolder| and its contents to \verb|$VSC_DATA| on
+the \hpc (we created some symbolic links in a previous section).
```

Comment:
Rephrase: `, assuming the 'data' symlink is present in your home directory, see section ... `